src/python/pants/option/global_options.py

```diff
@@ -435,6 +438,8 @@ def register_bootstrap_options(cls, register):
                   'and fall back to the local host if remote calls take longer than the speculation timeout.\n'
                   '`none`: Do not speculate about long running processes.',
              advanced=True)
+    register('--process-execution-use-local-cache', type=bool, default=True, advanced=True,
```

Comment:
I can see us having more `--process-execution`-prefixed flags. It would maybe be worth adding a TODO like:
```suggestion
    # TODO Make a subsystem to namespace `--process-execution`-prefixed flags.
    register('--process-execution-use-local-cache', type=bool, default=True, advanced=True,
```

Reply:
Also, I don't think it's relevant here, but passing `daemon=True` will make this option invalidate the daemon, in case that's something we want to do.

Reply:
We have this TODO further up in this file. But currently it's not possible to have a "bootstrap" subsystem.

> Also, I don't think it's relevant here, but passing daemon=True will make this option invalidate the daemon, in case that's something we want to do.

`daemon=True` is the default.

Reply:
We currently need this to be `daemon=True` (the default) because we statically create `CommandRunner`s early in the daemon lifecycle, and have no way to re-create them, or pass their config around dynamically. We _may_ want to change that at some point :) (cc @hrfuller)